services/grizzly-jersey/src/main/java/org/swecourse/services/Main.java

```diff
@@ -27,15 +27,15 @@
   private static final Optional < String > host;
   private static final Optional < String > port;
 
-  //public static Integer i = new Integer(123);
+  public static Integer i = new Integer(123);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this "Integer" constructor [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://159.89.108.87:9000/coding_rules#rule_key=squid%3AS2129)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this "public static i" field final [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://159.89.108.87:9000/coding_rules#rule_key=squid%3AS1444)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make i a static final constant or non-public and provide accessors if needed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://159.89.108.87:9000/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)